ChatSecure/Classes/Controllers/XMPP/OTRXMPPManager.m

```diff
@@ -1035,13 +1036,15 @@ - (void)parseSubscriptionFromRosterItem:(NSXMLElement *)item {
                 buddy.trustLevel = BuddyTrustLevelUntrusted;
                 buddy.displayName = [jid user];
                 buddy.username = [jid bare];
+            } else {
+                buddy = [buddy copy];
             }
             buddy.subscription = [SubscriptionAttributeBridge subscriptionWithString:subscription];
             [buddy setPendingApproval:[ask isEqualToString:@"subscribe"]];
```

Comment:
I thought pendingApproval could be readonly and calculated from the new enum cases we added?

Reply:
No, since we split the 9 states into two separate fields, subscription { none, from, to, both } and pending { none, out, in, outin }.

Reply:
Ah I see but it's still stored internally using the new enums. I was confused and thought we still had this old bool floating around.